### ISSUE TITLE
Question about clip_nll loss. 

### ISSUE THREAD

**Issue by YuGuiwe** [qiuqiangkong/panns_transfer_to_gtzan#9]
I am confused about the calculation clip_nll loss.
If my predict :
clipwise_output = [-1.2 ,  -2.3, -0.5]
target = [0, 0, 1]
torch.mean(clipwise_output  * target) = torch.mean([0, 0, -0.5]) = -0.166

Cross entropy was averaged by the number of category ?
Thanks for your reading...Hope it's not stupid question, I am beginner in training model.
